.gitignore

```diff
@@ -20,5 +20,23 @@
 bin
 bundle
 
-source/documentation/**/**/css
-source/documentation/**/**/js
+source/documentation/2.14/**/css
+source/documentation/2.14/**/js
+source/documentation/2.99/**/css
+source/documentation/2.99/**/js
+source/documentation/3.0/**/css
+source/documentation/3.0/**/js
+source/documentation/3.1/**/css
+source/documentation/3.1/**/js
+source/documentation/3.2/**/css
+source/documentation/3.2/**/js
+source/documentation/3.3/**/css
+source/documentation/3.3/**/js
+source/documentation/3.4/**/css
+source/documentation/3.4/**/js
+source/documentation/3.5/**/css
+source/documentation/3.5/**/js
+source/documentation/3.6/**/css
+source/documentation/3.6/**/js
+source/documentation/3.7/**/css
+source/documentation/3.7/**/js
```

Comment:
A comment here about why we ignore versions through 3.7 would be nice.

Reply:
Added but I'd forgotten to push to this branch, its on `source` if you want to read